### PR TITLE
Added smoke tests

### PR DESCRIPTION
**Pull request by nitinprabhu**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RPE-484

### Change description ###

- Added smoke tests.Implemented category annotation as some functional tests can be reused for smoke tests.

- Tested on sandbox.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/smokeTest/java/uk/gov/hmcts/reform/feature/TmpHealthCheckSmokeTest.java [hunk 1]
```diff
@@ -1,16 +0,0 @@
...
-public class TmpHealthCheckSmokeTest extends SmokeTestSuite {
-
-    @Test
-    public void should_get_success_from_health_endpoint() {
-        getCommonRequestSpec()
-            .get("/health")
-            .then()
-            .statusCode(OK.value());
-    }
-}
```
Comment (doncem) [inline src/smokeTest/java/uk/gov/hmcts/reform/feature/TmpHealthCheckSmokeTest.java]: 🎉

File: src/functionalTest/java/uk/gov/hmcts/reform/feature/FeatureToggleTest.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.feature;
+
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import uk.gov.hmcts.reform.feature.categories.SmokeTestCategory;
+
+import java.io.IOException;
+import java.util.UUID;
+
+public class FeatureToggleTest extends BaseTest {
```
Comment (rkondratowicz) [inline src/functionalTest/java/uk/gov/hmcts/reform/feature/FeatureToggleTest.java]: should that be `UpdateFeatureToggleTest`? (the other test class is called `CreateFeatureToggleTest`)

File: src/functionalTest/resources/feature-toggle-enabled.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "description": "Create feature for smoke test",
```
Comment (rkondratowicz) [inline src/functionalTest/resources/feature-toggle-enabled.json]: Is this a description of the action or the actual toggle?

File: src/functionalTest/java/uk/gov/hmcts/reform/feature/FeatureToggleTest.java (hunk 2 above)
Comment (nitinprabhu) [reply src/functionalTest/java/uk/gov/hmcts/reform/feature/FeatureToggleTest.java]: no it's enabling/disabling feature toggle not updating the properties of feature toggle.

File: src/functionalTest/resources/feature-toggle-enabled.json (hunk 3 above)
Comment (nitinprabhu) [reply src/functionalTest/resources/feature-toggle-enabled.json]: description of feature toggle

File: src/functionalTest/java/uk/gov/hmcts/reform/feature/FeatureToggleTest.java (hunk 2 above)
Comment (rkondratowicz) [reply src/functionalTest/java/uk/gov/hmcts/reform/feature/FeatureToggleTest.java]: 'change feature toggle state' test then?

Comment (rkondratowicz) [reply src/functionalTest/java/uk/gov/hmcts/reform/feature/FeatureToggleTest.java]: or 'switch feature toggle' test. ATM you've basically got `CreateFooBarTest` and `FooBarTest` which is confusing

**Pull request merged**